### PR TITLE
docs: update development instructions for testing the examples

### PR DESCRIPTION
**Pull request by gianpaj**

### CODE REVIEW

**Comment by gianpaj** [comment]
Actually, I had an error running `react-native run-ios` with the first example.

Should I open another issue about that @DylanVann ?

File: docs/development.md [hunk 1]
```diff
@@ -5,31 +5,31 @@ For now this uses a modified cli to work around issues with symlinked packages.
 This is how to start the example app so you can test code with it.
 
 ```bash
-# In the repo root folder.
-# Install dependencies.
+# In the repo root folder, install dependencies. This will also install SDWebImage submodules.
```
Comment (DylanVann) [inline docs/development.md]: There's actually no submodule anymore so this is slightly outdated.

File: docs/development.md [hunk 2]
```diff
@@ -5,31 +5,31 @@ For now this uses a modified cli to work around issues with symlinked packages.
...
-yarn link
+# Symlink the package folder during development.
+$ yarn link
 
-# Move to example folder.
-cd ReactNativeFastImageExample
+# Move to one of the example folders.
+cd react-native-fast-image-example/
+cd react-native-fast-image-example-cocoapods/
+cd react-native-fast-image-example-server/
```
Comment (DylanVann) [inline docs/development.md]: The following steps don't make sense for the server example.

**Comment by DylanVann** [comment]
The development docs have been updated more recently than this so it doesn't seem needed. Thank you!

**Pull request closed without merging**